CONTRIBUTORS.md

```diff
@@ -3,3 +3,4 @@
 - [@augb](https://github.com/augb)
 - [Daniel Elero](https://github.com/danixeee)
 - [yorodm](https://github.com/yorodm)
+- [Max O'Cull](https://github.com/Maxattax97)
```

Comment:
Contributors should be ordered alphabetically (https://github.com/openlawlibrary/pygls/blob/6e16f41da50ce0dcd83499d2d45adcb8dd10717d/CONTRIBUTORS.md#contributors-alphabetical).

Reply:
Can't believe I missed that :sweat_smile: